internal/models/user.go

```diff
@@ -382,6 +382,12 @@ func (u *User) Confirm(tx *storage.Connection) error {
 		return err
 	}
 
+	if err := u.UpdateUserMetaData(tx, map[string]interface{}{
```

Comment:
UpdateUserMetaData will only update user metadata field. How about identity_data in identity table?

Reply:
The identity data is updated in verify.go inside the signupVerify method, I thought this was the appropriate place as the identity data is updated in the verifySms method, not in the user. To keep things somewhat consistent